packages/react-ui/src/Accordion/index.js

```diff
@@ -19,11 +18,11 @@ export const Accordion = ({ items, noTitle, variant, ...props }) => {
     <AccordionVariant allowZeroExpanded allowMultipleExpanded {...props}>
       {items.map(({ body, icon, id, title }, index) => (
         <div id={id} key={`${id}-${index}`}>
-          {typeof id !== "undefined" &&
```

Comment:
not needed anymore

Reply:
why ?

Reply:
the header is not fixed anymore (except if we scroll up)